collection-renderer-lib/src/cmr/collection_renderer/services/collection_renderer.clj

```diff
@@ -32,6 +42,19 @@
     (.eval jruby (io/reader bootstrap-rb))
     jruby))
 
+(defn- get-preview-gem-umm-version
+  "Get the UMM schema version that is defined in preview gem."
+  []
+  (try
+    (-> preview-gem-umm-version-config-file
+        io/resource
+        slurp
+        str/trim)
+    (catch Exception e
+      (error "Failed to parse preview gem UMM version, will use the default version:"
+             default-preview-gem-umm-version
+             (.printStackTrace e)))))
```

Comment:
I put that in while working with MMT to settle down on the interface and it was good to not cause CMR to go belly up because the gem doesn't do the config correctly as we are now working off the current repo of MMT project. I agree with your comment, especially after move to versioned gems. I will take this try-catch block out.